schema.sql

```diff
@@ -0,0 +1,19 @@
+DROP DATABASE IF EXISTS bundle;
+
+CREATE DATABASE bundle;
+
+USE bundle;
+
+DROP TABLE IF EXISTS bundles;
+
+CREATE TABLE bundles (
+  id TINYINT(1) auto_increment,
+  name VARCHAR(68),
+  value TINYINT(1) UNSIGNED,
+  logoURL VARCHAR(2000),
+  helper VARCHAR(16),
+  minimum TINYINT(1) UNSIGNED,
+  PRIMARY KEY(id)
+);
+
+ALTER TABLE bundles AUTO_INCREMENT =1;
```

Comment:
I believe line 19, the ALTER TABLE line is starting the AUTO_INCREMENT at 1. That's great because we all decided to start the ID's at 1. My only suggestion for this file is to include a space after the equals sign on line 19. Other than that looks great.

Reply:
Will do.